react-ui/src/components/LogInForm.js

```diff
@@ -34,73 +34,93 @@ class logInForm extends Component {
     });
   };
 
-
   login = event => {
     let email = this.state.loginParams.email;
     let password = this.state.loginParams.password;
-    var params = {email: email, password: password}
-    axios.get(`/public/login/?${queryString.stringify(params)}`)
-    .then(res => {
-      if(res.data!==""){
-        this.setState({
-          email : res.data.email,
-          userType: res.data.userType,
-          firstName: res.data.firstName,
-          personId: res.data.workerId || res.data.studentId,
-          accessToken: res.data.accessToken,
-          islogged: true
+    let validinput = true;
```

Comment:
Hi Kyle,
For variable naming consistency, I would recommend updating this variable to lowerCamelCase, e.g., `validInput`.

Reply:
wording has been updated